invenio_app_ils/acquisition/loaders/jsonschemas/order.py

```diff
@@ -77,9 +77,11 @@ class Meta:
     order_date = DateString(required=True)
     order_lines = fields.List(fields.Nested(OrderLineSchema), required=True)
     payment = fields.Nested(PaymentSchema)
+    pid = PersistentIdentifier()
```

Comment:
`pid` is not needed here and all the others because it is automatically added by the parent class `RecordMetadataSchemaJSONV1`